services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/FacetTransformerRange.java

```diff
@@ -74,7 +74,7 @@ private SqlQuery generateFacetSqlQuery(SqlQuery originalQuery) {
 		SqlElementUntils.appendCombinedWhereClauseToStringBuilder(builder, facetSearchConditionString, tableExpressionFromModel.getWhereClause());
 		
 		try {
-			return new SqlQueryBuilder(builder.toString(), originalQuery.getTableSchema()).build();
+			return new SqlQueryBuilder(builder.toString(), originalQuery.getTableSchema(), 1L).build();

```

Comment:
You can use the originalQuery.getUserId() (once you add the getter), definetly not 1L :)